Data/ByteString/Builder/Internal.hs

```diff
@@ -414,6 +417,12 @@ instance Monoid Builder where
   {-# INLINE mconcat #-}
   mconcat = foldr mappend mempty
 
+instance IsList Builder where
+  type Item Builder = Word8
+  fromList = byteStringInsert . fromList
+  fromListN n = byteStringThreshold n . fromList
```

Comment:
Having a variable threshold is very strange. That means we will always run `byteStringCopy`, unless the provided list size is wrong. Why not just `byteStringCopy` or `byteString`?

Reply:
I am unsure which of the two would be a better choice here.

Reply:
Either is fine.